substrate/client/transaction-pool/src/fork_aware_txpool/mod.rs

```diff
@@ -378,6 +381,20 @@ mod view_store;
 
 pub use fork_aware_txpool::{ForkAwareTxPool, ForkAwareTxPoolTask};
 
+/// Code representing a case of a failure during transaction validation. It can be used
+/// as a code for a custom invalid transaction error.
+pub const RUNTIME_API_ERROR_WHILE_VALIDATING_CUSTOM_CODE: u8 = 1;
```

Comment:
Create a dedicated sub-module to contain them. Include also the `0` code used in the `TestApi`.